web/themes/custom/usagov/templates/html.html.twig

```diff
@@ -61,6 +61,11 @@
       {{'homepageTest'|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{front|json_encode(constant('JSON_PRETTY_PRINT'))|raw }},
       {{'basicPagesubType'|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{pagetype|json_encode(constant('JSON_PRETTY_PRINT'))|raw }},
       {{'Page_Type'|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{datalayer_pagetype|json_encode(constant('JSON_PRETTY_PRINT'))|raw }},
+      {{'hasBenefitCategory'|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}: {{datalayer_hasBenefitCategories|json_encode(constant('JSON_PRETTY_PRINT'))|raw }}
```

Comment:
Need a comma at the end of this line.

Reply:
good catch.  

Could we refactor how this is done in the future? Doing it in the template seems messy, seems like you could do the same with a preprocess hook to set the `$datalayer` variable (would have to be an object or array) and then have one line in twig to render it to JSON.

Reply:
Yes, I think that's a good idea.

Reply:
i'll file a jira ticket